lms/djangoapps/gating/signals.py

```diff
@@ -21,10 +19,13 @@ def handle_score_changed(**kwargs):
     Returns:
         None
     """
-    course = modulestore().get_course(CourseKey.from_string(kwargs.get('course_id')))
+    course = kwargs['course']
     if course.enable_subsection_gating:
+        subsection_grade = kwargs['subsection_grade']
+        new_score = subsection_grade.graded_total.earned / subsection_grade.graded_total.possible * 100.0
```

Comment:
Good point.  However, because it's 100-based, it's clients/users must have already submitted their minimum requirements with that in mind.  So I believe we need to continue to support it with the 100-base.